docs/2_auth.md

```diff
@@ -80,6 +80,8 @@ Note: The user is checked against the group members list on initial authenticati
    --client-secret=<value from step 6>
 ```
 
+Note: When using the Azure Auth provider with nginx and the cookie session store you may find the cookie is too large and doesn't get passed through correctly. Increasing the proxy_buffer_size in nginx or implementing the redis session storage (see https://pusher.github.io/oauth2_proxy/configuration#redis-storage) should resolve this.
```

Comment:
You might also mention that they might need to increase large_client_header_buffers to nginx

Reply:
I don't think large_client_header_buffers is needed for this change. The default in nginx for this is 8kb (https://nginx.org/en/docs/http/ngx_http_core_module.html#large_client_header_buffers) and the cookie resulting from this change is under 8kb. It's above 4kb though which is mostly the default for proxy_buffer_size (https://nginx.org/en/docs/http/ngx_http_proxy_module.html#proxy_buffer_size).